client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -0,0 +1,178 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { noop } from 'lodash';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { AspectRatios } from 'state/ui/editor/image-editor/constants';
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'EditGravatar', function() {
```

Comment:
Sorry, I didn't bring this up earlier, but our component test docs are severely out of date. We now generally try to use shallow [enzyme](https://github.com/airbnb/enzyme) renders to test.

A good recent example of a component test can be seen at #9150 Ping me with any extra questions